bin/docker-compose

```diff
@@ -81,6 +81,12 @@ function __main__() {
     NGINX_ENABLED="false"
   fi
 
+  if [ -z ${SHARELATEX_LISTEN_IP+x} ]; 
```

Comment:
`SHARELATEX_LISTEN_IP=''` will not work for the bind mount, `:80:80` is invalid.

> ports contains an invalid type, it should be a number, or an object

WDYT about merging the case into the "not set" case?

```suggestion
  if [ -z ${SHARELATEX_LISTEN_IP} ]; 
```

Reply:
> WDYT about merging the case into the "not set" case?

Sorry, not sure what you mean by this?

Reply:
> > WDYT about merging the case into the "not set" case?
> 
> Sorry, not sure what you mean by this?

Consider the "variable set to empty value" case like the "variable not set" case and use the default value for both.